app/presenters/contact_presenter.rb

```diff
@@ -48,7 +48,6 @@ def contact_details
       slug: contact.slug,
       title: contact.title,
       description: contact.description,
-      organisation: ContactOrganisationPresenter.new(contact.organisation).present,
```

Comment:
Uuuh. We'd been relatively blissfully ignorant of contact groups, 'cause it seems like they're an admin construct, not actually used anywhere on the frontend. I've made a note to look at them.